### PR TITLE
workload: add pprof handler to `workload csv-server`

### PR DESCRIPTION
**Pull request by danhhz**
Release note: None

### CODE REVIEW

File: pkg/cmd/workload/csv_server.go [hunk 1]
```diff
@@ -49,6 +50,7 @@ func runCSVServer(_ *cobra.Command, _ []string) error {
 			}
 		})
 	}
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
```
Comment (danhhz) [inline pkg/cmd/workload/csv_server.go]: I just added this one, should I do all of them?

Comment (jordanlewis) [reply pkg/cmd/workload/csv_server.go]: Might as well add the memory one as well. The others don't seem as important.

**Comment by danhhz** [comment]
RFAL

---

Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[pkg/cmd/workload/csv_server.go, line 53 at r1](https://reviewable.io:443/reviews/cockroachdb/cockroach/23434#-L6sIN7N80llSlmy6_Og-r1-53:-L6sKBzmnKLjQGzjo_X3:b-3fq4wa) ([raw file](https://github.com/cockroachdb/cockroach/blob/428c930d379099e829a95298b14126f5f15996c8/pkg/cmd/workload/csv_server.go#L53)):*
<details><summary><i>Previously, jordanlewis (Jordan Lewis) wrote…</i></summary><blockquote>

Might as well add the memory one as well. The others don't seem as important.
</blockquote></details>

Turns out heap is actually handled by the Index one, which I found so surprising I decided to add all of them just in case. Clearly I don't know what's important and what's not

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/cockroach/23434#-:-L6sKMoBh568DVo85pNR:bb72pd0)*

**Review by jordanlewis — APPROVED** [review]
👍

**Pull request merged**